gleam.toml

```diff
@@ -11,8 +11,8 @@ links = [
 ]
 
 [dependencies]
-gleam_stdlib = ">= 0.34.0 and < 2.0.0"
-gleam_json = ">= 1.0.0 and < 2.0.0"
+gleam_stdlib = ">= 0.44.0 and < 2.0.0"
+gleam_json = ">= 2.0.0 and < 3.0.0"
```

Comment:
Leave the lower bound at v1 at the code still supports that please

Reply:
@lpil when using the current version of `json-typedef` that uses `gleam_json` v1.0.1 I try to run `gleam check` and I get an error regarding the deprecated `gleam/string_builder` package and it makes me unable to run my code. Even if the code supports the v1 package what's the point if people can't build their projects since versions below 2.1.0 use a deprecated package?

Reply:
That's because you're using a newer stdlib. It won't be the case for people using older versions, and we have no reason to exclude them here. Constraints should always be as relaxed as possible to avoid annoying conflicts.

This package will be upgraded to use new features, at which point the version constraint will need to be tightened.